doc/articles/Ownership.md

```diff
@@ -24,6 +24,14 @@ Note that such functions will *not* free any previously-allocated data: overwrit
 
 Note also that some structs with `FreeMembers` functions may be used as both inputs and outputs. In this case `FreeMembers` must only be used if the member allocations were made by the `webgpu.h` implementation (as an output), not if they were made by the application (as an input).
 
+## Releasing a Device object {#DeviceRelease}
+
+Unlike other destroyable objects, releasing the last (external) ref to a device causes it to be automatically destroyed (via @ref wgpuDeviceDestroy) if it isn't already lost or destroyed. Though the device object is no longer valid to use after releasing the last ref, this has some notable direct effects:
+
+- It destroy buffers, unmapping them, aborting any pending map requests, and preventing future ones.
```

Comment:
nit. This reads a bit weird because we are effectively saying it destroys buffers, which means unmapping/aborting/preventing.

Maybe:
"It destroys buffers which will unmap them, abort any pending map requests, and prevent future map requests."